test/core/TheClog.t.sol

```diff
@@ -78,6 +78,22 @@ contract TooDeepRecursionClogTest is MangroveTest, IMaker {
     takeSome(minVolume);
     takeSome(minVolume + 1);
   }
+
+  function testFail_take_more_than_first_fails_for_deep_stack() public {
```

Comment:
`testFail` will be [soon](https://github.com/foundry-rs/foundry/pull/4969) deprecated/discouraged by foundry so I think we could move this test and the one below to a "normal" test?

Reply:
done